src/opera/instance/node.py

```diff
@@ -131,7 +131,16 @@ def get_attribute(self, params):
         if attr in self.attributes:
             return self.attributes[attr].eval(self)
 
-        # TODO(@tadeboro): Add capability access.
+        # If we have no attribute, try searching for capability.
+        capabilities = tuple(
+            c for c in self.template.capabilities if c.name == attr
+        )
+        if len(capabilities) > 1:
```

Comment:
I see, this can be a problem. Now I've added additional checks to find a requirement and a capability with the same names and when one of them is used in `get_attribute` function opera throws an error.